resources/server/calls/calls.service.ts

```diff
@@ -222,6 +223,7 @@ class CallsService {
     // lets protect against that
     if (currentCall?.is_accepted) {
       emitNet(CallEvents.WAS_ENDED, currentCall.receiverSource);
+      emitNet(CallEvents.WAS_ENDED, currentCall.transmitterSource);
     }
     // player who is calling (transmitter)
     resp({ status: 'ok' });
```

Comment:
Is there any point to this being a promiseMessageEvent if it isn't transactional? If we wish to go with classic events and their fire and forget behavior, then lets just make this a regular event handler.

Reply:
I would assume 

> The developer who implemented this part of the phone (shame on that Taso guy) should have explicitly defined all the possible states each participant can be in and made a nice map of definitions but this wasn't done.
> 
> I'll see if I get all the possible states and outcomes mapped out, so we can be real _developers_

Would just be best if we had a simple util for `getPhoneState(playerId)`. We could probably use statebags for this actually I think. Not sure if that’s something you would want to add to the phone but we can do a state for inCall, and inPendingCall. And then just check the states of the player when someone is calling them and set it to whatever on the corresponding call events.

Reply:
I just threw in some events without looking at the rest of it 😆
Definitely need some changes.

Reply:
If it 'definitely need some changes', lets make sure those are done, before merging. Rather fix it now, instead of having to go back and fix it again.